File: Assets/Scripts/boom.js
var up : boolean = false;
var playing : boolean = false;

function FixedUpdate(){
	
	if(!playing){
		GetComponent.<AudioSource>().Play();
		playing = true;
	}
	if (!up){
		transform.position.y += 0.1;
		up = true;
	}else{
		transform.position.y -= 0.1;
		up = false;
	}
}

File: Assets/Scripts/_boom.cs
using UnityEngine;

public class _boom : MonoBehaviour {
	bool up = false;
	bool playing = false;

	void FixedUpdate() {
		if (!playing) {
			GetComponent<AudioSource>().Play();
			playing = true;
		}
		
		Vector3 pos = transform.position;
		if (!up) {
			pos.y += 0.1f;
			transform.position = pos;
			up = true;
		} else {
			pos.y -= 0.1f;
			transform.position = pos;
			up = false;
		}
	}
}
